quark/Evaluator/pyeval.py

```diff
@@ -333,10 +333,10 @@ def AGET_OBJECT(self, instruction):
         try:
 
             array_obj = self.table_obj.get_obj_list(
-                int(instruction[2][1]),
```

Comment:
if `instrucion[2] = v12` then `instruction[2][1] = 1` but it should be correct 12